### PR TITLE
Ported the ACE2 MX-2A thermal imaging device

### PR DESCRIPTION
**Pull request by ulteq**

### CODE REVIEW

**Comment by Glowbal** [comment]
Do we want to keep this in an MX2A pbo or combine this with a few other bino's that we could port from ACE2?

**Comment by alganthe** [comment]
why not put it into thermals ?

**Comment by ulteq** [comment]
> why not put it into thermals ?
Thermals is not about items/weapon. It's about changing how thermal imaging looks in general.

File: addons/mx2a/stringtable.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <Czech>MX-2A</Czech>
+            <Russian>MX-2A</Russian>
+            <Italian>MX-2A</Italian>
+            <Polish>MX-2A</Polish>
+            <French>MX-2A</French>
+            <Hungarian>MX-2A</Hungarian>
+        </Key>
+        <Key ID="STR_ACE_MX2A_Description">
+            <English>Thermal Scope</English>
+            <German>Thermal Scope</German>
```
Comment (esteldunedain) [inline addons/mx2a/stringtable.xml]: The non-translated entries should be deleted (Spanish in particular, but afaik also Italian and French)

**Comment by esteldunedain** [comment]
:+1: Remember to always ask for confirmation before porting p3d files from ACE2, as we can't release some of them under GPL.

**Pull request merged**